07-systems_access_policy.md

```diff
@@ -41,7 +41,7 @@ Access to Health Note systems and applications is limited for all users, includi
 ## 7.2 Access Establishment and Modification
 
 1. Requests for access to Health Note systems and applications are made formally using the following process:
-  1. A Health Note workforce member initiates the access request by filling out a **Systems Access Form** [here](https://docs.google.com/forms/d/e/1FAIpQLSdus2wN2WHjbj0JBal2aG6Z8NmguTP7am1GCmZAgPvb7zSKaw/viewform?usp=pp_url&entry.768761467=Pending).
+  1. A Health Note workforce member initiates the access request by filling out a [**Systems Access Form**](https://docs.google.com/forms/d/e/1FAIpQLSdus2wN2WHjbj0JBal2aG6Z8NmguTP7am1GCmZAgPvb7zSKaw/viewform?usp=pp_url&entry.768761467=Pending).
      * User identities must be verified prior to granting access to new accounts.
      * Identity verification must be done in person where possible; for remote employees, identities must be verified over the phone.
      * For new accounts, the method used to verify the user's identity must be recorded on the Systems Access Form.
```

Comment:
For new accounts, we are doing things through Rippling and Roles. Wonder if we should update that here as well.

Reply:
Yes, we should update to our current process to include Rippling, though for changes, it seemed we are still using the Google form.  We can distinguish access establishment via Rippling for new users.

Reply:
@jameschu2550 to take this on.

Reply:
Added bullet point for Rippling